### PR TITLE
Small doc fixes

### PR DESCRIPTION
**Pull request by smoreface**
Updates to make some info more findable
* database setup page title update
* cross links for image annotation units

### CODE REVIEW

File: docs/source/guide/storage.md [hunk 1]
```diff
@@ -309,15 +309,14 @@ Add these variables to your environment setup:
...
-
 ### Set up connection in the Label Studio UI
 In the Label Studio UI, do the following to set up the connection:
 
 1. Open Label Studio in your web browser.
 2. For a specific project, open **Settings > Cloud Storage**.
 3. Click **Add Source Storage**.  
 4. In the dialog box that appears, select **Local Files** as the storage type. 
-5. Specify **Local path** directory with your files. The local path must include the LOCAL_FILES_DOCUMENT_ROOT in the beginning. For example, if `LOCAL_FILES_DOCUMENT_ROOT=/home/user`, then your local path must be `/home/user/dataset1`.
+5. Specify **Local path** directory with your files. The local path must be an absolute path and include the LOCAL_FILES_DOCUMENT_ROOT. For example, if `LOCAL_FILES_DOCUMENT_ROOT=/home/user`, then your local path must be `/home/user/dataset1`. For more about that environment variable, see [Run Label Studio on Docker and use local storage](start.html#Run_Label_Studio_on_Docker_and_use_local_storage)
```
Comment (niklub) [inline docs/source/guide/storage.md:319]: Is 1 at the end of `dataset1` is a typo, or was it intentional?

File: docs/source/guide/predictions.md [hunk 2]
```diff
@@ -8,9 +8,9 @@ meta_description: Label Studio Documentation for importing predicted labels, pre
 
 If you have predictions generated for your dataset from a model, either as pre-annotated tasks or pre-labeled tasks, you can import the predictions with your dataset into Label Studio for review and correction. Label Studio automatically displays the pre-annotations that you import on the Labeling page for each task. 
 
-To import predicted labels into Label Studio, you must use the [Basic Label Studio JSON format](tasks.html#Basic-Label-Studio-JSON-format) and set up your tasks with the `predictions` JSON key. The Label Studio ML backend also outputs tasks in this format. 
+To import predicted labels into Label Studio, you must use the [Basic Label Studio JSON format](tasks.html#Basic-Label-Studio-JSON-format) and set up your tasks with the `predictions` JSON key. The Label Studio ML backend also outputs tasks in this format. You must use different IDs for each task elements, annotations, predictions and their `result` items.
```
Comment (niklub) [inline docs/source/guide/predictions.md:11]: actually, you don't need to specify IDs when importing tasks with preannotations.

**Review by niklub — CHANGES_REQUESTED** [review]

File: docs/source/guide/storage.md (hunk 1 above)
Comment (smoreface) [reply docs/source/guide/storage.md:319]: It's intentional, also used in Tasks with local storage file references section

File: docs/source/guide/predictions.md (hunk 2 above)
Comment (smoreface) [reply docs/source/guide/predictions.md:11]: Ah okay cool, this was a holdover from the previous docs

Comment (smoreface) [reply docs/source/guide/predictions.md:11]: Removed

**Review by niklub — APPROVED** [review]

**Pull request merged**